tasks/__init__.py

```diff
@@ -1003,14 +1003,14 @@ def set_maintenance(start=None, end=None):
         invoke set_maintenance_state --start 2016-03-16T15:41:00-04:00
         invoke set_maintenance_state --end 2016-03-16T15:41:00-04:00
     """
-    set_maintenance(start, end)
+    set_maintenance(ctx, start, end)
```

Comment:
This is not an invoke task; it is an imported function. Make sure that `ctx` is only added to calls to invoke tasks.